x-pack/plugins/entity_manager/server/lib/entities/built_in/services_from_ecs_data.ts

```diff
@@ -65,72 +50,7 @@ export const builtInServicesFromEcsEntityDefinition: EntityDefinition =
       'cloud.provider',
       'cloud.availability_zone',
       'cloud.machine.type',
-    ],
-    metrics: [
-      {
-        name: 'latency',
-        equation: 'A',
-        metrics: [
-          {
-            name: 'A',
-            aggregation: 'avg',
-            filter: serviceTransactionFilter(),
-            field: 'transaction.duration.histogram',
-          },
-        ],
-      },
-      {
-        name: 'throughput',
-        equation: 'A',
-        metrics: [
-          {
-            name: 'A',
-            aggregation: 'value_count',
-            filter: serviceTransactionFilter(),
-            field: 'transaction.duration.summary',
-          },
-        ],
-      },
-      {
-        name: 'failedTransactionRate',
-        equation: '1 - (A / B)',
-        metrics: [
-          {
-            name: 'A',
-            aggregation: 'sum',
-            filter: serviceTransactionFilter(),
-            field: 'event.success_count',
-          },
-          {
-            name: 'B',
-            aggregation: 'value_count',
-            filter: serviceTransactionFilter(),
-            field: 'event.success_count',
-          },
-        ],
-      },
-      {
-        name: 'logErrorRate',
-        equation: 'A',
-        metrics: [
-          {
-            name: 'A',
-            aggregation: 'doc_count',
-            filter:
-              'log.level: "error" OR log.level: "ERROR" OR error.log.level: "error" OR error.log.level: "ERROR"',
-          },
-        ],
-      },
-      {
-        name: 'logRate',
-        equation: 'A',
-        metrics: [
-          {
-            name: 'A',
-            aggregation: 'doc_count',
-            filter: 'data_stream.type: logs',
-          },
-        ],
-      },
+      'kubernetes.namespace',
+      'orchestrator.cluster.name',
```

Comment:
Just noting that this is then only for ECS, not SemConv, we'll need to follow up on that but I think we can cover that as we look at the upcoming Kubernetes entities.

Reply:
Yes, just had a chat with @roshan-elastic and I'm adding the oTel ones now.